src/styles/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+  line-height: 16px;
+}
+
+.stars__star {
+  box-sizing: border-box;
+  list-style-type: none;
+  background: url(../images/star.svg) no-repeat center center;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background: url(../images/star-active.svg) no-repeat center center;
+}
```

Comment:
The way you're handling active stars isn't scalable. What if you have different amounts of active stars? Instead of creating a new class for each possible amount of active stars, consider a solution that would work for any number of active stars. You could, for instance, add a modifier class to the stars that should be active.